data/moves.js

```diff
@@ -17576,8 +17572,7 @@ let BattleMovedex = {
 			return null;
 		},
 		onBasePower(basePower, pokemon, target) {
-			if (pokemon.hasItem('utilityumbrella')) return;
-			if (this.field.isWeather(['raindance', 'primordialsea', 'sandstorm', 'hail'])) {
+			if (this.field.isWeather(pokemon.hasItem('utilityumbrella') ? ['sandstorm', 'hail'] : ['raindance', 'primordialsea', 'sandstorm', 'hail'])) {
```

Comment:
I don't really like how long this line is, maybe changing it to a variable? it functions the same either way but using a separate function has more readability